src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+
+  Learn more about browser [here](/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic browser"
+  alt="New Relic browser"
+  src={NRBrowserMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Dashboards [#dashboards]
+<SideBySide>
+  <Side>
+
+  <img
+  title="New Relic dashboards"
+  alt="New Relic dashboards"
+  src={NRDashboardsMain}
+/>
+
+  </Side>
+  <Side>
+
+  Our UI allows you to create custom charts and share them easily. Our <InlinePopover type="dashboards" /> capability lets you collect and visualize your data preferences across our platform. Tailor your charts to display the information you need. Access and analyze your data from any location within our platform.
+
+  NRQL, short for New Relic query language, is a powerful SQL-like tool for accessing detailed data from New Relic. It lets you gather specific info about your applications, hosts, and business operations. By using NRQL, you can gain valuable insights into the performance and behavior of your environment. 
+
+  Learn more about dashboards [here](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) and about NRQL [here](/docs/nrql/get-started/introduction-nrql-new-relics-query-language/).
+  
+  </Side>
+  </SideBySide>
+
+# Errors inbox [#errors-inbox]
+<SideBySide>
+  <Side>
+
+  Errors inbox is an error-tracking solution designed to put you in control. It helps you find, focus on, and fix errors across your application stack. Errors inbox is embedded in the <InlinePopover type="apm" /> UI so you can resolve errors faster.
+
+  Learn more about errors inbox [here](/docs/apm/errors-inbox/errors-inbox-ui).
+
+  </Side>
+  <Side>
+
+  <img
+  title="New Relic Errors inbox"
+  alt="New Relic Errors inbox"
+  src={NRErrosInboxMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+# IAST [#iast]
+<SideBySide>
+  <Side>
+
+  <img
+  title="New Relic IAST"
+  alt="New Relic IAST"
+  src={NRIastMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic Interactive Application Security Testing (IAST) can help you stop cyberattacks and breaches on your applications. It does this by probing your running code for exploitable flaws. Use IAST to cut blind spots, see all protected and unprotected applications, and hide threats. Our dashboards are comprehensive and give complete visibility into high-priority vulnerabilities. 
+
+  Learn more about IAST [here](/docs/iast/introduction).
+
+  </Side>
+  </SideBySide>
+
+# Infrastructure [#infrastructure]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="infrastructure" /> capability lets you monitor the health and performance of your service, including containers and cloud infrastructure. It covers tracking CPU, memory, network traffic, and disk use. It also includes over 400 on-host integrations for monitoring third-party apps.
```

Comment:
I think this should be unspooled more, and I'm not sure SREs/devOps personas necessarily associate infrastructure with services so much as containers/cloud infra/traditional server infra. Maybe going to a higher level to describe how infrastructure monitoring can tell you about CPU/memory _and_ related to kubernetes/on host integrations/cloud monitoring... It's a tough section to write about but I do think it should be a little longer/unspooled

Reply:
or maybe they care about services insofar as CPU/memory/disk space/blahblah is there for the purpose of maintaining app availability. but i would def speak directly to the people who only care about the infrastructure and only reference services/apps as an incidental thing

Reply:
As I added in a previous comment, Austin and Shwan don't want to have a lot of text for every capability.